pkg/kubelet/dockertools/manager.go

```diff
@@ -52,6 +53,9 @@ const (
 	podOomScoreAdj = -100
 
 	maxReasonCacheEntries = 200
+
+	kubernetesPodLabel       = "io.kubernetes.pod"
+	kubernetesContainerLabel = "io.kubernetes.container"
```

Comment:
We already encode pod name as "io.kubernetes.pod.name" can we use "io.kubernetes.container.name" here?

WDYT of "io.kubernetes.pod.data" or something along those lines for the above?